pkg/exchange/okex/convert.go

```diff
@@ -255,20 +251,27 @@ func toLocalOrderType(orderType types.OrderType) (okexapi.OrderType, error) {
 	return "", fmt.Errorf("unknown or unsupported okex order type: %s", orderType)
 }
 
-func toGlobalOrderType(orderType okexapi.OrderType) (types.OrderType, error) {
+func toGlobalOrderType(orderType okexapi.OrderType) types.OrderType {
 	switch orderType {
-	case okexapi.OrderTypeMarket:
-		return types.OrderTypeMarket, nil
+	case okexapi.OrderTypeMarket, okexapi.OrderTypeMarketMakerProtection:
+		return types.OrderTypeMarket
+
 	case okexapi.OrderTypeLimit:
-		return types.OrderTypeLimit, nil
+		return types.OrderTypeLimit
+
 	case okexapi.OrderTypePostOnly:
-		return types.OrderTypeLimitMaker, nil
+		return types.OrderTypeLimitMaker
 
-	case okexapi.OrderTypeFOK:
-	case okexapi.OrderTypeIOC:
+	case okexapi.OrderTypeIOC, okexapi.OrderTypeFOK:
+		return types.OrderTypeMarket
```

Comment:
It's a bit weird to make ioc/fok orders as market orders. @c9s, what do you think to make these two types `types.OrderTypeLimitMaker`?

Reply:
yeah, IOC or FOK doesn't mean Market order type, you might need to check some other property.

Reply:
ok, IOC/FOK I change to `types.OrderTypeLimitMaker`